tests/templates/kuttl/authorizer/03-install-druid.yaml.j2

```diff
@@ -11,8 +11,8 @@ metadata:
   name: derby-druid
 spec:
   image:
-    productVersion: "{{ test_scenario['values']['druid'].split('-stackable')[0] }}"
-    stackableVersion: "{{ test_scenario['values']['druid'].split('-stackable')[1] }}"
+    productVersion: "{{ test_scenario['values']['druid-latest'].split('-stackable')[0] }}"
+    stackableVersion: "{{ test_scenario['values']['druid-latest'].split('-stackable')[1] }}"
```

Comment:
This currently saves one test. The druid-opa-authorizer version is coupled to respective druid versions, i would keep the test for all versions.

Reply:
ok, reverted.